packages/migrate/src/__tests__/DbPush.test.ts

```diff
@@ -88,7 +88,7 @@ describeIf(process.platform !== 'win32')('push', () => {
       Prisma schema loaded from prisma/schema.prisma
       Datasource "my_db": SQLite database "dev.db" at "file:dev.db"
 
-      SQLite database dev.db created at file:dev.db
+      sqlite database dev.db created at file:dev.db
```

Comment:
Hard to avoid this "regression"? Just 2 lines above it is correctly `SQLite`...

Reply:
I'll have a look, it was not a top priority thing compared to the other changes

Reply:
Fixed